Frameworks/SPMySQLFramework/Source/SPMySQLConnection.m

```diff
@@ -961,8 +1009,10 @@ - (BOOL)_reconnectAfterBackgroundConnectionLoss
  */
 - (BOOL)_waitForNetworkConnectionWithTimeout:(double)timeoutSeconds
 {
+
+    SPLog(@"_waitForNetworkConnectionWithTimeout: %f", timeoutSeconds);
 	// Set up the reachability target - the host is not important, and is not connected to.
-	SCNetworkReachabilityRef reachabilityTarget = SCNetworkReachabilityCreateWithName(NULL, "dev.mysql.com");
+    SCNetworkReachabilityRef reachabilityTarget = SCNetworkReachabilityCreateWithName(NULL, "dev.mysql.com"); //FIXME: is this the best to use?
```

Comment:
Hmmmm maybe we could try google.com or something?

Reply:
I read somewhere you're not supposed to check for a connection to a host. Apple updated their ancient reachability code recently to show this. I'll switch to Google for now.